helm/app/templates/networkPolicies/metabase-ingress.yaml

```
@@ -16,3 +16,24 @@ spec:
       podSelector:
         matchLabels:
           app.kubernetes.io/instance: {{ .Values.metabase.instanceName }}
+
+{{- if .Values.metabase.prodIngress.enable }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-metabase-prod-ingress
+  labels:
+{{ include "ccbc.labels" . | nindent 4 }}
```

```suggestion
  labels: {{ include "ccbc.labels" . | nindent 4 }}
```
The point of `nindent` is that you can define it on the previous line. This way you don't need to break the indentation in your template, and it's more readable